src/api/v3/locations.py

```diff
@@ -0,0 +1,68 @@
+from django.http import JsonResponse
+
+from cms.models import POI
+from ..decorators import json_response
+
+
+def transform_poi_translation(poi_translation):
+    """
+    Function to create a JSON from a single poi_translation object.
+
+    :param poi_translation: The poi translation object which should be converted
+    :type poi_translation: ~cms.models.pois.poi_translation.POITranslation
+
+    :return: return data necessary for API
+    :rtype: dict
+    """
+
+    # is there a better way to access this?
+    try:
+        icon_url = poi_translation.icon.url
+    except AttributeError:
+        icon_url = ""
+
+    poi = poi_translation.poi
+    return {
+        "id": poi_translation.id,
+        "url": poi_translation.permalink,
+        "path": poi_translation.slug,
+        "title": poi_translation.title,
+        "modified_gmt": poi_translation.last_updated,
```

Comment:
This does not match the exact format required by https://github.com/Integreat/cms/wiki/REST-APIv3-Documentation#pageseventslocationsdisclaimer. However, other parts of the api send the `modified_gmt` in the exact same way. Should I change this to match the exact format?